Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
@DomT4  Sure, but I think "what package the library comes from" is not relevant to enabling the LZMA compression method. Brew metadata abstracts that as intended.

But if the consensus is `xz` then that's what I'll use.